h/api/search/__init__.py

```diff
@@ -77,21 +77,13 @@ def includeme(config):
         name='legacy_es',
         reify=True)
 
-    # request.new_es is always a client for the legacy Elasticsearch index,
-    # regardless of whether the 'postgres' feature flag is on.
-    # This should be used to write to the new search index.
-    # TODO: Remove when postgres migration is done
-    config.add_request_method(
-        lambda r: _get_client(r.registry.settings),
-        name='new_es',
-        reify=True)
-
     # request.es is a client for either the new or the legacy Elasticsearch
```

Comment:
I've removed this comment and moved the code to where the old comment was. So once we can get rid of all legacy code we just need to remove the block where it adds the `legacy_es` method.